frontend/router/index.tsx

```diff
@@ -140,6 +146,10 @@ const routes = (
             </Route>
             <Route path="integrations/mdm/windows" component={WindowsMdmPage} />
             <Route path="integrations/mdm/apple" component={MacOSMdmPage} />
+            <Route
+              path="integrations/automatic-enrollment/windows"
```

Comment:
@noahtalerman, this route seems odd to me. I see that this is the intended route in [Figma](https://www.figma.com/file/hdALBDsrti77QuDNSzLdkx/%F0%9F%9A%A7-Fleet-EE%2C-June-2023-and-earlier?type=design&node-id=18756-226545&mode=design&t=Bu8ro61ytHCvhDs8-0), but my initial impression was that this should follow the pattern `integrations/mdm/windows/automatic-enrollment. Additionally, it seems at odds with our usage guidelines for turning on/off features like MDM vs. "enrollment". Perhaps something like "Identity and access management" might be a good way to refer to collection of settings related to ABM, Autopilot, Azure AD, end-user auth (IdP), and EULA? (FWIW that's how Azure AD is described by [Wikipedia](https://en.wikipedia.org/wiki/Microsoft_Azure_Active_Directory) and it seems like broad enough category to encompass the others too.)

Reply:
@gillespi314 hmm, the route is driven by the intended UI/UX. To get to the `integrations/automatic-enrollment/windows` route, the user clicks to navigate to the **Integrations > Automatic enrollment > Windows** page.

The same thinking applies to `integrations/mdm/apple`. User clicks to navigate to the **Integrations > MDM > Apple Push Certificates** page.

Regarding enrollment vs MDM on/off, we're using enrollment here because these are the integrations involved when trying to get a host to automatically show up in Fleet when it's unboxed. While the features also automatically turn on MDM, the main goal here is to have the host show up in Fleet (enroll) automatically.

In addition, we want the page to represent the use case: automatic enrollment. Other tools use "Identity and access management" which describes the suite of tools rather than the use case.

Reply:
@noahtalerman 

Gotcha. If that is the rationale, I think I would be helpful to refine the messaging to distinguish the MDM tab from the automatic enrollment tab. 

In dogfood at least, it seems like the page-level description should be updated. Currently, it says "Add ticket destinations and turn on mobile device management features."

And a tab-level description for automatic enrollment should be added. Perhaps something like "Configure settings to enable devices to automatically enroll to Fleet."

Current dogfood screenshot:
<img width="1060" alt="Screenshot 2023-07-28 at 10 40 35 AM" src="https://github.com/fleetdm/fleet/assets/73313222/d8093b9c-1d8f-42c6-bbe3-57c131066516">

Reply:
> the page-level description should be updated. Currently, it says "Add ticket destinations and turn on mobile device management features."

Ah, good idea. Currently, this page level description doesn't change based on which tab you're on (ticket destinations v. automatic enrollment) so I don't think we want to show "Configure settings to enable devices to automatically enroll to Fleet."

Knowing this, what do you think we should change the description to?

> the current text for ABM is jumbled

What should we change it to?

Reply:
> > the page-level description should be updated. Currently, it says "Add ticket destinations and turn on mobile device management features."
> 
> Ah, good idea. Currently, this page level description doesn't change based on which tab you're on (ticket destinations v. automatic enrollment) so I don't think we want to show "Configure settings to enable devices to automatically enroll to Fleet."
> 
> Knowing this, what do you think we should change the description to?

Maybe: "Add ticket destinations, turn on mobile device management features, and configure automatic enrollment settings." 

> 
> > the current text for ABM is jumbled
> 
> What should we change it to?

Depends on how you want to say it, but there's two verbs. I'd probably just remove "use".

Reply:
I'd probably invert the order of the tabs so they flow top-to-bottom: 
Automatic enrollment
MDM
Ticket destinations

The the description could be: "Configure settings to enable devices to automatically enroll to Fleet, turn on mobile device management features, and add ticket destinations"